medusa/network_timezones.py

```diff
@@ -67,17 +67,26 @@ def update_network_dict():
     network_list = dict(cache_db_con.select('SELECT * FROM network_timezones;'))
 
     queries = []
-    for network, timezone in iteritems(d):
+    remote_network_list = iteritems(d)
```

Comment:
Ok then just leave the `for network, timezone in remote_network_list:` And add the log with the return at line 65.